includes/overrides.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * This file contains hooks and functions that override the behavior of WP Core.
+ *
+ * @package TenUpScaffold
+ */
+
+namespace TenUpScaffold\Overrides;
+
+/**
+ * Registers instances where we will override default WP Core behavior.
+ *
+ * @link https://developer.wordpress.org/reference/functions/print_emoji_detection_script/
+ * @link https://developer.wordpress.org/reference/functions/print_emoji_styles/
+ * @link https://developer.wordpress.org/reference/functions/wp_staticize_emoji/
+ * @link https://developer.wordpress.org/reference/functions/wp_staticize_emoji_for_email/
+ * @link https://developer.wordpress.org/reference/functions/wp_generator/
+ * @link https://developer.wordpress.org/reference/functions/wlwmanifest_link/
+ * @link https://developer.wordpress.org/reference/functions/rsd_link/
+ *
+ * @return void
+ */
+function setup() {
+	// Remove the Emoji detection script.
+	remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
+
+	// Remove inline Emoji detection script.
+	remove_action( 'admin_print_scripts', 'print_emoji_detection_script' );
+
+	// Remove Emoji-related styles from front end and back end.
+	remove_action( 'wp_print_styles', 'print_emoji_styles' );
+	remove_action( 'admin_print_styles', 'print_emoji_styles' );
+
+	// Remove Emoji-to-static-img conversion.
+	remove_filter( 'the_content_feed', 'wp_staticize_emoji' );
+	remove_filter( 'comment_text_rss', 'wp_staticize_emoji' );
+	remove_filter( 'wp_mail', 'wp_staticize_emoji_for_email' );
+
+	add_filter( 'tiny_mce_plugins', __NAMESPACE__ . '\disable_emojis_tinymce' );
+	add_filter( 'wp_resource_hints', __NAMESPACE__ . '\disable_emoji_dns_prefetch', 10, 2 );
+
+	// Remove WordPress generator meta.
+	remove_action( 'wp_head', 'wp_generator' );
+	// Remove Windows Live Writer manifest link.
+	remove_action( 'wp_head', 'wlwmanifest_link' );
+	// Remove the link to Really Simple Discovery service endpoint.
+	remove_action( 'wp_head', 'rsd_link' );
+
+
+}
+
+/**
+ * Filter function used to remove the TinyMCE emoji plugin.
+ *
+ * @link https://developer.wordpress.org/reference/hooks/tiny_mce_plugins/
+ *
+ * @param  array $plugins An array of default TinyMCE plugins.
+ * @return array          An array of TinyMCE plugins, without wpemoji.
+ */
+function disable_emojis_tinymce( $plugins ) {
```

Comment:
Just poking around here - is this still applicable with the 5.0 release of Gutenberg?

Reply:
Gutenberg has its own emoji script, so this should only be relevant to TinyMCE, specifically instances of the classic editor.